pyproject.toml

```diff
@@ -15,7 +15,6 @@ classifiers = [
     "Intended Audience :: End Users/Desktop",
     "Intended Audience :: Financial and Insurance Industry",
     "Intended Audience :: Information Technology",
-    "License :: OSI Approved :: MIT License",
```

Comment:
Is this redundant?

Reply:
Yes, https://peps.python.org/pep-0639/#deprecate-license-classifiers